lib/core/rancher/step.sh

```diff
@@ -120,6 +133,18 @@ function ih::setup::core.rancher::install() {
       fi
     fi
 
+    # Use vz (requires macOS >=13.3) instead of qemu on M3 macs to resolve issues.
+    # More details: https://github.com/lima-vm/lima/issues/1996
+    if ih::arch::is_m3_mac; then
```

Comment:
Planning to only make this switch for m3 mac users for now, since vz is still "experimental" according to the docs. I think a few of the platform team people are going to try using it on non-M3 macs and see if it breaks anything.